WordPress/Classes/WordPress.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>WordPress 29.xcdatamodel</string>
+	<string>WordPress 30.xcdatamodel</string>
```

Comment:
We've already bumped the data model version for 5.3. Is it necessary to bump again?  I think we've been aiming for once per release.